### PR TITLE
Cleanup of service api

### PR DESCRIPTION
**Pull request by hendrikebbers**
Removed several methods that are not needed

### CODE REVIEW

**Review by iwsimon — APPROVED** [review]
LGTM

**Review by Neeharika-Sompalli — APPROVED** [review]

**Review by povolev15 — DISMISSED** [review]
LGTM

File: hedera-node/hedera-app-spi/src/main/java/com/hedera/node/app/spi/Service.java [hunk 1]
```diff
@@ -20,18 +20,10 @@
...
  */
 public interface Service {
-    /**
-     * Returns the name of the service. This name must be unique for each service deployed on the
-     * application.
-     *
-     * @return the name
-     */
-    @NonNull
-    String getServiceName();
```
Comment (netopyr) [inline hedera-node/hedera-app-spi/src/main/java/com/hedera/node/app/spi/Service.java:34]: I don't think it makes much sense to remove this method right now. We may have to put it back quite soon or replace it with very similar functionality.

**Review by netopyr — APPROVED** [review]
The part in `hedera-app-spi` LGTM.

Not sure though, if these changes make sense right now. I think quite soon, we will be in a position where we can define how services should be added dynamically. At that point, this whole area has to be adjusted anyway.

**Review by nathanklick — CHANGES_REQUESTED** [review]
Merge conflicts and failing E2E tests.

**Comment by mhess-swl** [comment]
Merge conflict and failing integration test run

**Review by timo0 — APPROVED** [review]

**Review by povolev15 — APPROVED** [review]

**Review by nathanklick — APPROVED** [review]
Approval limited to the `module-info.java` changes only. Please seek other reviews.

**Comment by mhess-swl** [comment]
Merge conflicts need resolved

**Pull request closed without merging**